acq4/devices/PatchPipette/states.py

```diff
@@ -1041,16 +1041,18 @@ class PatchPipetteResealState(PatchPipetteState):
     ----------
     initialPressure : float
         Initial pressure (Pa) to apply (default is -0.5 kPa)
-    maximumPressure : float
+    maxPressure : float
         Maximum pressure (Pa) to apply (default is -4 kPa)
     pressureChangeRate : float
         Rate at which pressure should change during reseal (default is -0.5 kPa / min)
     retractionSpeed : float
-        Speed in m/s to move pipette during retraction (default is 0.3 um / s)
+        Speed in m/s to move pipette during retraction (default is 0.3e-6 (0.3 µm / s))
     resealTimeout : float
         Seconds before reseal attempt exits
     numTestPulseAverage : int
         Number of test pulses to average when measuring resistance
+    slowDetectionThreshold : float
+        Threshold for slow change in pipette resistance (Ohm) to trigger cell detection (default 200 kOhm)
```

Comment:
To trigger reseal detection?

Reply:
Maybe? I didn't actually grok the code, I just noticed it was used at all but not included in the defaults or docs.